protocol/testutil/keeper/prices.go

```diff
@@ -123,7 +123,7 @@ func CreateNMarkets(t *testing.T, ctx sdk.Context, keeper *keeper.Keeper, n int)
 	numExistingMarkets := GetNumMarkets(t, ctx, keeper)
 	for i := range items {
 		items[i].Param.Id = uint32(i) + numExistingMarkets
-		items[i].Param.Pair = fmt.Sprintf("%v", i)
+		items[i].Param.Pair = fmt.Sprintf("%v-%v", i, i)
```

Comment:
why do we do this?

Reply:
The assumption I'm making here is that pairs in the dydx market params are formated `BASE-QUOTE` based on https://github.com/skip-mev/v4-chain/blob/main/protocol/x/prices/testdata/expected_default_genesis.json

Whereas our currency pair strings are `BASE/QUOTE`. 

The keeper adapter will just ignore (and log) anything which doesn't conform to this convention.